<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fishing Organizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="sessionTable"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnectDatabase"/>
    <addaction name="actionConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddBait"/>
    <addaction name="actionAddFish"/>
    <addaction name="actionAddLocation"/>
    <addaction name="actionAddCatch"/>
    <addaction name="actionAddSession"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewBait"/>
    <addaction name="actionViewFishes"/>
    <addaction name="actionViewLocations"/>
    <addaction name="actionViewCatches"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionAnalysisYear"/>
    <addaction name="actionAnalysisWaters"/>
    <addaction name="actionZonalDistribution"/>
   </widget>
   <widget class="QMenu" name="menuMaintenance">
    <property name="title">
     <string>Maintenance</string>
    </property>
    <addaction name="actionConsistencyCheck"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuMaintenance"/>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit forg without saving</string>
   </property>
  </action>
  <action name="actionAddBait">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Bait</string>
   </property>
  </action>
  <action name="actionViewBait">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View Bait</string>
   </property>
  </action>
  <action name="actionConnectDatabase">
   <property name="text">
    <string>Connect Database</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionViewFishes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View Fishes</string>
   </property>
  </action>
  <action name="actionViewLocations">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View Locations</string>
   </property>
  </action>
  <action name="actionAddFish">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Fish</string>
   </property>
  </action>
  <action name="actionAddLocation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Location</string>
   </property>
  </action>
  <action name="actionAddSession">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Session</string>
   </property>
  </action>
  <action name="actionViewSession">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View Session</string>
   </property>
  </action>
  <action name="actionViewCatches">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View Catches</string>
   </property>
  </action>
  <action name="actionAddCatch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Catch</string>
   </property>
  </action>
  <action name="actionAnalysisYear">
   <property name="text">
    <string>Analysis of the year</string>
   </property>
  </action>
  <action name="actionAnalysisWaters">
   <property name="text">
    <string>Waters specific</string>
   </property>
  </action>
  <action name="actionZonalDistribution">
   <property name="text">
    <string>Zonal distribution of catches</string>
   </property>
  </action>
  <action name="actionConsistencyCheck">
   <property name="text">
    <string>Consistency check</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sessionTable</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>addButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
